src/data/equiv/basic.lean

```diff
@@ -336,6 +336,12 @@ lemma perm_congr_symm_apply {α β : Type*} (e : α ≃ β) (p : equiv.perm β)
 protected lemma image_eq_preimage {α β} (e : α ≃ β) (s : set α) : e '' s = e.symm ⁻¹' s :=
 set.ext $ assume x, set.mem_image_iff_of_inverse e.left_inv e.right_inv
 
+@[simp] lemma symm_preimage_eq_image {α β} (e : α ≃ β) (s : set α) : e.symm ⁻¹' s = e '' s :=
+by symmetry; exact equiv.image_eq_preimage e s
+
+lemma symm_image_eq_preimage {α β} (e : α ≃ β) (s : set β) : e ⁻¹' s = e.symm '' s :=
```

Comment:
The name of the lemma is wrong, the current statement is called `preimage_eq_symm_image`.  But maybe it would be better to flip the equation and mark it simp (see my other comment).

Reply:
```suggestion
@[simp] lemma symm_image_eq_preimage {α β} (e : α ≃ β) (s : set β) : e.symm '' s = e ⁻¹' s :=
```